Makefile

```diff
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := all
 
 VIRTUALENV := build/virtualenv
-PYTHON_VERSION := python3
+PYTHON_VERSION := python3.8
```

Comment:
Any reason why our default is now 3.8? (I'm fine with that btw)

Reply:
Check #224 and #228

Reply:
Lol - it was my suggestion to keep 3.8.... 🙈